### PR TITLE
show pending deposit status

### LINKED ISSUE #128: show deposits countdown
## Scope
- we delay deposits in the node by 8 main-net blocks
- so in bridge-ui, when a deposit is submitted to main-net, there should be a countdown
- the countdown counts the blocks, until the 8 blocks passed (around that time the node will include the plasma deposit) (edited) 

xdai chain has a good example:
screen shot 2019-01-22 at 10 34 02 am

- we have this screen on the top right which is showing the tx in progress (approval, deposit). maybe just extend that one.
- it will be confusing for users without the countdown, because the plasma balance would still be 0, but main-net balance decreased already.

## Deliverables
- bridge-ui pr

## Alignment with purpose
- better ux for chain

## Bounty

gardener: @johannbarbie | 10%
worker: @sunify | 80%
reviewer: ?? | 10%

### PR DESCRIPTION
**Pull request by troggy**
Resolves #128

Depends on https://github.com/leapdao/leap-node/pull/165, https://github.com/leapdao/leap-core/pull/64

### CODE REVIEW

File: src/routes/wallet/deposit.tsx [hunk 1]
```diff
@@ -39,14 +58,77 @@ export default class Deposit extends React.Component<DepositProps, any> {
...
+    this.checkPendingDeposits();
+  }
+
+  @autobind
+  private loadPendingDeposits() {
+    const { color } = this.props;
+    this.pendingDeposits = storage.load(this.storageKey).filter(dep => dep.color === color);
+  }
+
+  private storePendingDeposits() {
```
Comment (sunify) [inline src/routes/wallet/deposit.tsx]: You can use reactions to call this method

Comment (sunify) [reply src/routes/wallet/deposit.tsx]: Or even better — use `autorun`

Comment (troggy) [reply src/routes/wallet/deposit.tsx]: Reaction is not working for me 🤔

```
reaction(() => this.pendingDeposits, this.storePendingDeposits);
```

never reacts. Tried to add comparator as well

Comment (troggy) [reply src/routes/wallet/deposit.tsx]: autorun works though 🎉

**Pull request merged**